.circleci/config.yml

```diff
@@ -28,6 +28,7 @@ executors:
     docker:
       - image: klaytn/build_base:1.1-go1.14.6-solc0.4.24
       - image: localstack/localstack:0.10.9
+      - image: circleci/mysql:5.7
```

Comment:
The test will connect with the `root:root@/?parseTime=True`.
Doesn't it need to add an environment parameter for MYSQL password? something like `MYSQL_ROOT_PASSWORD`: root.

Reply:
We could setup password as you said, but since it is same as username and it's only for test purposes, what about test without password?

Reply:
Sure. I think it is fine.